packages/xml-views-validation/src/utils/is-binding-attribute-value.ts

```diff
@@ -0,0 +1,7 @@
+export function isPossibleBindingAttributeValue(value: string): boolean {
+  // Check if an attribute value might be a binding expression.
+  // The current check is very naive - it only looks for matching curly brackets anywhere in the value.
```

Comment:
maybe nitpicking on description but this regExp will find the longest substring surrounded by curly brackets (its a greedy match) not **matching** (you mean balanced?) brackets.

Reply:
I think we shouldn't check the syntax of the binding expression (like if the brackets are balanced) and consider anything with `{` and `}` (in this order) as a possible binding expression that shouldn't be validated. I don't do anything with the result expect check if it matches so it doesn't matter if it's greedy.

Reply:
I'm fine with the logic, just meant that the description in the comment is more like look for a "pair" of curly brackets than "matching" (assuming matching implies balanced). thats why I said this comment may be nitpicking :)

Reply:
Ah, ok. I didn't realize you were talking about the comment.